capmvm/kubernetes/Dockerfile

```diff
@@ -1,20 +1,36 @@
-FROM ghcr.io/weaveworks-liquidmetal/flintlock-ubuntu-base:20.04
-
+ARG OS_VERSION=20.04
```

Comment:
(no changes required, only a note)

This can be `UBUNTU_VERSION` or something that tells it's ubuntu, `OS_VERSION` would require me to open the `Dockerfile` and see "what os, heh?". Maybe we could just use
```dockerfile
ARG BASE_OS="ubuntu:20.04"
FROM ${BASE_OS}
```

but then, all apt commands would fail if it's not a debian based base image :/

Reply:
In the meantime, would you settle from some very good documentation (in another pr)?

Reply:
That can work too :)